Package.swift

```diff
@@ -4,18 +4,27 @@ import PackageDescription
 
 let package = Package(
   name: "meilisearch-swift",
+  platforms: [
+    .iOS(.v12),
+    .tvOS(.v12),
+    .watchOS(.v5),
+    .macOS(.v10_15)
+  ],
```

Comment:
I had to add these `platforms` values because of the `jwt` dependency. But yes, we draw the line of which platforms we will support doing this.

- macOS https://gs.statcounter.com/macos-version-market-share/desktop/worldwide (we are supporting the big market)
- ios https://developer.apple.com/support/app-store/ as well (more than 98%)
- tvOS and watchOS I was not able to acquire data about them.

Reply:
BTW, we could consider as a `breaking-change` because of this.

Reply:
I think it is a good idea!